Description: Move locale dir to /usr/share/fvwm/locale to avoid conflicts with fvwm3.
Author: Jaimos Skriletz <jaimosskriletz@gmail.com>
Forwarded: not-needed
Last-Updated: 2021-05-31
--- a/configure.ac
+++ b/configure.ac
@@ -1274,7 +1274,7 @@ AC_SUBST(FVWMTASKBAR_DOMAIN)
 AC_SUBST(FVWMSCRIPT_DOMAIN)
 AC_SUBST(ALL_DOMAINS)
 
-LOCALEDIR='${datadir}'"/locale"
+LOCALEDIR="$FVWM_DATADIR/locale"
 with_gettext="yes"
 problem_gettext=""
 
